SinaiStCatherine/SinaiNewEt002.xml

```diff
@@ -207,33 +392,47 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                          <q xml:lang="gez">ለዕመ ዐልብየ፡ ዝንቱመጸሀ<gap reason="lost"/> ዐልብየ፡ ተናዝዞት፡ ካልዕ</q>
                        </item>
                        <item xml:id="a6">
-                         <locus target="#110v"/>
-                         <desc type="Unclear">Note in the upper margin.</desc>
-                         <q xml:lang="gez">ሀይጣሕል፡ ዝንቱ፡ መጸሐፈ፡ ንሳዕዐ</q>
+                         <locus target="#64v"/>
+                         <desc type="Unclear">Written in the upper margin in red ink.</desc>
+                         <q xml:lang="gez">ቶማስ፡</q> <!--create person ID?-->
                        </item>
                        <item xml:id="a7">
                          <locus target="#110v"/>
-                         <desc type="Unclear">Note in the lower margin, possibly continuing <ref target="#a6"/>.</desc>
-                         <q xml:lang="gez">ዐበጊ፡ ዘወተር፡ ሀይጣሕል፡ ከተከከ<gap reason="lost"/>ክርስቶስ፡ ሀተታዐምር</q>
+                         <desc type="Unclear">Note in the upper margin, partly incomprehensible.</desc>
```

Comment:
I would like to have a look at the notes, I would need the numbers of images, to come through recto - verso sets :)